src/app/clusters/tls-certificate-management-server/CertificateTableImpl.cpp

```diff
@@ -722,3 +753,28 @@ CHIP_ERROR CertificateTableImpl::GetClientCertificateCount(FabricIndex fabric, u
 {
     return mClientCertificates.GetFabricEntryCount(fabric, outCount);
 }
+
+inline CHIP_ERROR SwallowNotFound(CHIP_ERROR err)
+{
+    return (err == CHIP_ERROR_NOT_FOUND) ? CHIP_NO_ERROR : err;
+}
+
+CHIP_ERROR CertificateTableImpl::RemoveFabric(FabricIndex fabric)
+{
+    CHIP_ERROR result = SwallowNotFound(mClientCertificates.RemoveFabric(fabric));
+    ReturnErrorOnFailure(SwallowNotFound(mRootCertificates.RemoveFabric(fabric)));
```

Comment:
This could use some documentation explaining what the goal is (looks like making sure we call RemoveFabric on both cert arrays, but then only touching storage bits if both of those removals succeeded?

Reply:
Thanks, good call out, added doc & made that a bit clearer